jwst/stpipe/step.py

```diff
@@ -24,7 +24,8 @@
 
 SUFFIX_LIST = ['rate', 'cal', 'uncal', 'i2d', 's2d', 's3d',
                'jump', 'ramp', 'x1d', 'x2d', 'x1dints', 'calints', 'rateints']
```

Comment:
There isn't an "x2d" suffix used anywhere. We should probably add "crf" and "crfints" for completeness.